pkg/osquery/tables/gsettings/gsettings.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const gsettingsPath = "/usr/bin/gsettings"
 
-const allowedCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-."
+const allowedCharacters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789_-. "
```

Comment:
This can't be changed in isolation. Pretty sure:
* #L101 will need better quoting
* #L161 may or may not be okay

This is going to need a manual test pass to ensure it's works as designed. Since there's some shell in here, anything with spaces feels risky